package.json

```diff
@@ -4,12 +4,20 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
```

Comment:
I added these by typing this into this part of the package.json file.

This corresponds to running this command to install the packages:

`npm i express get-port`

and 

`npm i -D prettier`